src/Route.jsx

```diff
@@ -60,6 +60,22 @@ export function getRoutesLayer(geojson) {
   }));
 }
 
+export function getSubRoutesLayer(subroute) {
+  const subrouteLayer = new PathLayer({
+    id: 'path-layer',
+    data: {
```

Comment:
this should be `[ ...subroute, ]`, not `{...subroute,}` (as the error we're getting is props.data.map does not exist, which implies that it was expecting a list)